client/src/components/CardContainer/CardContainer.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {NavLink} from "react-router-dom";
+import {Swiper, SwiperSlide} from "swiper/react";
+import {Navigation} from 'swiper'
+import CardItem from '../CardItem/CardItem';
+import styles from './CardContainer.module.scss';
+
+
+import "swiper/css";
+import "swiper/css/pagination";
+import "swiper/css/navigation";
+
+
+const cards = [
+    <CardItem/>,
+    <CardItem/>,
+    <CardItem/>,
+    <CardItem/>,
+    <CardItem/>,
+    <CardItem/>,
+    <CardItem/>,
+    <CardItem/>,
+    <CardItem/>,
+    <CardItem/>,
+    <CardItem/>,
+    <CardItem/>,
+]
+
+
+const CardContainer = () => {
+
+    return (
+        <div className={styles.cardContainer}>
+            <div className={styles.headerContainer}>
+                <h2 className={styles.header}>Popular tours</h2>
+                <NavLink className={styles.allTours} to="/catalogue">View All</NavLink>
+            </div>
+            <div className={styles.swiperContainer}>
+                <div id='previous' className={styles.btnPrev}></div>
+                <div id='next' className={styles.btnNext}></div>
+                <Swiper
+                    breakpoints={{
+                        265: {
+                            width: 265,
```

Comment:
Please, delete all width properties in breakpoints as there are no necessity in them